Automattic/wp-calypso
# Sites Management Page: Visually distinguish 'redirect' sites

Fixes #66746

## Proposed Changes

Visually distinguishes 'redirect' sites in the List view and Grid view, and uses `site.options.unmapped_url` instead of `site.URL` for redirect sites:

<img width="1422" alt="image" src="https://user-images.githubusercontent.com/36432/185612146-46e64f02-fe64-4519-8de2-356cb0004985.png">

<img width="1398" alt="image" src="https://user-images.githubusercontent.com/36432/185612252-fb1b9791-0978-4051-a099-b18f2523de0b.png">

## Testing Instructions

1. Create a new Free site.
2. Add the Site Redirect Add-On to the new Free site: https://wordpress.com/domains/add/site-redirect/
3. Navigate to `/sites-dashboard`.
4. Verify the Free site with redirect appears as expected in both List view and Grid view.
